docs/topics/annotations.md

```diff
@@ -169,27 +169,150 @@ class Example {
 The full list of supported use-site targets is:
 
   * `file`
-  * `property` (annotations with this target are not visible to Java)
   * `field`
+  * `property` (annotations with this target are not visible to Java)
   * `get` (property getter)
   * `set` (property setter)
+  * `all` (an experimental meta-target for properties, see [below](#all-meta-target) for its purpose and usage)
   * `receiver` (receiver parameter of an extension function or property)
+
+    To annotate the receiver parameter of an extension function, use the following syntax:
+
+    ```kotlin
+    fun @receiver:Fancy String.myExtension() { ... }
+    ```
+    
   * `param` (constructor parameter)
   * `setparam` (property setter parameter)
   * `delegate` (the field storing the delegate instance for a delegated property)
 
-To annotate the receiver parameter of an extension function, use the following syntax:
+### Defaults when no use-site targets are specified
+
+If you don't specify a use-site target, the target is chosen according to the `@Target` annotation of the annotation
+being used.
+If there are multiple applicable targets, the first applicable target from the following list is used:
+
+* `param`
+* `property`
+* `field`
+
+For example:
 
 ```kotlin
-fun @receiver:Fancy String.myExtension() { ... }
+data class User(val username: String,
+                // @Email here is equivalent to @param:Email
+                @Email val email: String) {
+    // @Email here is equivalent to @field:Email
+    @Email val secondaryEmail: String? = null
+}
 ```
 
-If you don't specify a use-site target, the target is chosen according to the `@Target` annotation of the annotation
-being used. If there are multiple applicable targets, the first applicable target from the following list is used:
+Kotlin %kotlinEapVersion% introduced an experimental defaulting rule which should
```

Comment:
```suggestion
Kotlin 2.2.0 introduced an experimental defaulting rule which should
```
As this content should only be merged with the final release, it's best to mention the actual release version so the EAP version doesn't automatically update later.